src/macros.asm

```diff
@@ -27,6 +27,24 @@ MACRO enemy x, y, type
 	.db type, x << 4 | y
 ENDM
 
+;
+; LevelHeader macro
+;
+; The order of the parameters is slightly different than how it's encoded, but
+; hopefully this order is a little more intuitive?
+;
+MACRO levelHeader pages, horizontal, bgPalette, spritePalette, music, objectTypeAXFX, objectType3X9X, groundSetting, groundType
```

Comment:
@Xkeeper0 i took some liberties with the order here based on what felt readable in the level data. hope you like it, because it's gonna be a pain to rework it in all the level data again 😛

Reply:
Search and replace! With regex! It's not that hard, just dumb.

See an earlier comment w/r/t possibly duplicating the argument list as a comment in the level files.